### DISCUSSION TITLE
Run dependabot alerts on PRs too

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#21322] [Code Security] [answered]
I don't want to get an alert after merging a vulnerable dependency to the main branch but before right in the PR.

**Comment by rsclarke** [comment, marked as answer]
@hfhbd this is possible with the [Dependency review](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review) action.

**Reply by hfhbd** [reply]
Thanks, didn't know, I expected a built-in solution. But this action sounds nice, I will try it.
